src/pydata_sphinx_theme/__init__.py

```diff
@@ -864,42 +864,55 @@ def get_pygments_stylesheet(light_style, dark_style):
 
 def _overwrite_pygments_css(app, exception=None):
     """
-    Sphinx is not build to host multiple sphinx formatter and there is no way
-    to tell which one to use and when.
-    So yes, at build time we overwrite the pygment.css file so that it embeds
-    2 versions:
-    - the light theme prefixed with "[data-theme="light"]" using tango
-    - the dark theme prefixed with "[data-theme="dark"]" using monokai
-
-    When the theme is switched, Pygments will be using one of the preset css
-    style.
-    """
-    default_light_theme = "tango"
-    default_dark_theme = "monokai"
+    Overwrite pygments.css to allow dynamic light/dark switching.
```

Comment:
Would something like this work for all the Sphinx-based docs? I mean, adding this bit to allow for the dynamic switch?

If so I'd love to add this to the "mini-docs" of accessible-pygments

Reply:
I think so? I think we adapted it from the Furo theme.